src/lib/plugins/commands.js

```diff
@@ -335,6 +335,30 @@ module.exports.server = function (serv, { version }) {
     return serv.selector(type, data)
   }
 
+  serv.parseSelectorString = (str, pos, world, allowUser) => {
+    if (/^@/.test(str) && !(/^@([arpe])(?:\[([^\]]+)\])?$/.test(str))) throw new UserError('Unknown selector type')
+    if (!(/^@/.test(str))) { return { str: serv.getPlayer(str) } }
+    return serv.selectorString(str, pos, world, allowUser)
+  }
+  /**
+   * @description send to a user a prismarine-chat message
+   * @param {string} aboutUser user the message is about
+   * @param {boolean} isConsole executed by console (alias for showToUser to be true)
+   * @param {ChatMessage} message message sent to user
+   * @param {boolean} showToUser show the message to @var aboutUser
+   * @param {string} sender usually is the preset
+   * @param {int} position usually is the preset
+   */
+  serv.sendChatMessage = (aboutUser, isConsole, message, showToUser, sender = '00000000-0000-0000-0000-000000000000', position = 1) => {
```

Comment:
You can import the uuid library and do `uuid.v3()` to generate a random UUID here, otherwise you should define the default constant at the top of the file. Also, why the need for two parameters that do the same thing? The logic could be simplified here

Reply:
it's not random